### PR TITLE
feat(translation): add dutch translation

### LINKED ISSUE #232: Add Dutch translations
🇳🇱 

@RolfKoenders tagged this as assigned but please don't worry about working on this until whenever you like. Also please don't hesitate to not do this at all if you don't want to :)

Instructions to add a new language are [here](https://github.com/gitpoint/git-point/issues/233#issuecomment-321917690) :)

### PR DESCRIPTION
**Pull request by RolfKoenders**
Closes #232

_had to push with --no-verify_

### CODE REVIEW

File: src/auth/screens/language-settings.js [hunk 1]
```diff
@@ -7,4 +7,8 @@ export default [
     code: 'fr',
     name: 'Français',
   },
+  {
+    code: 'nl',
+    name: 'Dutch',
```
Comment (lex111) [inline src/auth/screens/language-settings.js]: @RolfKoenders there must be a native language name, if I'm not mistaken, then `Nederlands`, not `Dutch`?

Comment (RolfKoenders) [inline src/auth/screens/language-settings.js]: Correct!

File: src/locale/languages/nl.js [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+      title: 'Volgers',
+    },
+    followingList: {
+      title: 'Volgen',
+    },
+  },
+  repository: {
+    main: {
+      shareRepositoryTitle: 'Delen {{repoName}}',
+      shareRepositoryMessage: 'Check out {{repoName}} on GitHub. {{repoUrl}}',
```
Comment (lex111) [inline src/locale/languages/nl.js]: Here it seems not translated

Comment (RolfKoenders) [inline src/locale/languages/nl.js]: Correct I was in doubt if we should translate the share message?

File: src/locale/languages/nl.js [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+    repositoryList: {
+      title: 'Repositories',
+    },
+    followerList: {
+      title: 'Volgers',
+    },
+    followingList: {
+      title: 'Volgen',
+    },
+  },
```
Comment (lex111) [inline src/locale/languages/nl.js]: For #260 PR there is a [new phrase](https://github.com/gitpoint/git-point/pull/260/files#diff-f9a4a43ac6485b52db67745369cbcc2d) for translation, you can add it below now:
```js
followYou: {
 title: 'Follows you',
},
```

File: src/locale/languages/nl.js (hunk 2 above)
Comment (lex111) [inline src/locale/languages/nl.js]: If the `shareRepositoryTitle` is translated, then why not?

File: src/locale/languages/nl.js [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+        submitted: 'ingediend',
+        dismissed: 'dismissed',
+        published: 'gepubliceerd',
+        publicized: 'publicized',
+        privatized: 'privatized',
+        starred: 'heeft een ster gegeven aan',
+        pushedTo: 'pushed to',
+        forked: 'forked',
+        commented: 'heeft een opmerking gemaakt',
+        removed: 'removed',
```
Comment (lex111) [inline src/locale/languages/nl.js]: This is a general action (not specific), it can also be translated

File: src/locale/languages/nl.js [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+        unlabeled: 'unlabeled',
+        opened: 'geopend',
+        milestoned: 'milestoned',
+        demilestoned: 'demilestoned',
+        closed: 'gesloten',
+        reopened: 'heropend',
+        review_requested: 'review aangevraagd',
+        review_request_removed: 'review aangevraag verwijderd',
+        submitted: 'ingediend',
+        dismissed: 'dismissed',
```
Comment (lex111) [inline src/locale/languages/nl.js]: Not sure, here exactly?

File: src/locale/languages/nl.js [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+      issueType: 'Issue',
+      applyLabelButton: 'Voeg label toe',
+      noneMessage: 'geen',
+      labelsTitle: 'LABELS',
+      assignYourselfButton: 'Assign jezelf',
+      assigneesTitle: 'ASSIGNEES',
+      actionsTitle: 'ACTIES',
+      unlockIssue: 'Ontsluiten {{issueType}}',
+      lockIssue: 'Sluit {{issueType}}',
+      closeIssue: 'Sluit {{issueType}}',
```
Comment (lex111) [inline src/locale/languages/nl.js]: The same for lock and close?

**Pull request merged**